<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTask</class>
 <widget class="QMainWindow" name="AddTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Task</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Img/Img/icon.jpg</normaloff>:/Img/Img/icon.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Header">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Body">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="StartTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>185</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="ExpEndTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>245</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="Category">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>308</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>2</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Header</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>168</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>229</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>End time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Difficulty</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Difficulty">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>308</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="EndTimeError">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>228</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>End time &lt;= Start time!</string>
    </property>
   </widget>
   <widget class="QLabel" name="DescriptionError">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Field cannot be empty!</string>
    </property>
   </widget>
   <widget class="QLabel" name="HeaderError">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>2</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Field cannot be empty!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
